<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbDrink">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NextDrinkingTime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbEye">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NextEyeShiftTime:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDrinkInter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>喝水间隔（分钟）</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbEyeInter">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>远眺间隔（分钟）</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinDrink">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinEye">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
    <property name="singleStep">
     <number>25</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkMinimizeToTray">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭窗口时最小化到托盘</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHealthReminder">
    <property name="title">
     <string>HealthReminder</string>
    </property>
   </widget>
   <addaction name="menuHealthReminder"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
